HeterogeneousCore/Producer/test/testGPUMock_cfg.py

```diff
@@ -28,3 +28,6 @@
 )
 process.p = cms.Path(process.prod1+process.prod2)#+process.eca)
 #process.p.associate(process.t)
+
+# Example of forcing module to run a specific device for one module via configuration
+#process.prod1.heterogeneousEnabled_ = cms.untracked.PSet(force = cms.untracked.string("GPUMock"))
```

Comment:
is this still the chosen syntax ?

Reply:
As we discussed, the `force` parameter forces the module to be run on a given device, while the per-device-type flags enable/disable those device types. At the moment it is not very useful in practice though, as algorithms implementing CUDA versions are always scheduled there.